code/addons/vitest/src/postinstall.ts

```diff
@@ -80,7 +80,7 @@ export default async function postInstall(options: PostinstallOptions) {
             name: 'migrateToNextjsVite',
             message: dedent`
             The addon requires the use of @storybook/nextjs-vite to work with Next.js.
-            https://storybook.js.org/docs/writing-tests/test-addon#install-and-set-up
+            https://storybook.js.org/docs/next/${DOCUMENTATION_LINK}#install-and-set-up
```

Comment:
Should this use `api.getDocsUrl` (as done [here](https://github.com/storybookjs/storybook/blob/next/code/addons/vitest/src/components/GlobalErrorModal.tsx#L82-L86))?

Reply:
this is a node file and can't use the storybook API unfortunately